templates/shortcode-receipt.php

```diff
@@ -150,12 +150,12 @@
 						<div class="edd_purchase_receipt_product_name">
 							<?php echo esc_html( $item['name'] ); ?>
 							<?php if( ! is_null( $price_id ) ) : ?>
-							<span class="edd_purchase_receipt_price_name">&nbsp;&ndash;&nbsp;<?php echo edd_get_price_option_name( $item['id'], $price_id, $payment->ID ); ?></span>
+							<span class="edd_purchase_receipt_price_name">&nbsp;&ndash;&nbsp;<?php echo esc_attr( edd_get_price_option_name( $item['id'], $price_id, $payment->ID ) ); ?></span>
 							<?php endif; ?>
 						</div>
 
 						<?php if ( $edd_receipt_args['notes'] ) : ?>
-							<div class="edd_purchase_receipt_product_notes"><?php echo wpautop( edd_get_product_notes( $item['id'] ) ); ?></div>
+							<div class="edd_purchase_receipt_product_notes"><?php echo wp_kses_post(wpautop( edd_get_product_notes( $item['id'] ) ) ); ?></div>
```

Comment:
How do you use that variable?  I don't understand how to do that.  I tried this and tried with making the variable global at the top of the page but had no luck. 

```
<?php echo strip_tags( edd_get_product_notes( $item['id'] ), $allowedposttags ); ?>
```